src/webgpu/web_platform/copyToTexture/ImageBitmap.spec.ts

```diff
@@ -339,10 +336,7 @@ g.test('from_canvas')
           'rgba8unorm-srgb',
           'bgra8unorm-srgb',
           'rgb10a2unorm',
-          'rgba16float',
-          'rgba32float',
           'rg8unorm',
-          'rg16float',
         ] as const)
```

Comment:
nit: We have three copies of this list. We should deduplicate them into capability_info.ts so we can update them more easily later.

Reply:
One issue when I try to move the defination to capability_info.ts as const kValidTextureFormatForCopyE2T= [...] as const.
I couldn't use kValidTextureFormatForCopyE2T.include(format) (format is kAllTextureFormats) in the validation tests. The compile reports type mismatch assignment.
Do you have any suggestions?

Reply:
You could use something like `const kValidTextureFormatForCopyE2T: read-only GPUTextureFormat[] = ...` if that works (but maybe it will cause other issues, I don't know

If that doesn't work just put in `as` casts wherever needed to fix the problems.

Reply:
FYI, I discuss with my collegues working on TFJS for suggestions and we'd like to find a solution. So I'd like to hold the CL for one more day and use tricks `as` later if I cannot get some ideas.

Reply:
Finally adopt ([...object] as string[]).includes(...) :p